src/chain/network_event.rs

```diff
@@ -51,19 +50,6 @@ pub struct EventSigPayload {
 }
 
 impl EventSigPayload {
-    pub fn new<T: Serialize>(
-        key_share: &bls::SecretKeyShare,
-        payload: &T,
-    ) -> Result<Self, RoutingError> {
-        let sig_share = key_share.sign(&serialize(&payload)?[..]);
-        let pub_key_share = key_share.public_key_share();
-
-        Ok(Self {
-            pub_key_share,
-            sig_share,
-        })
-    }
-
     pub fn new_for_section_key_info(
```

Comment:
maybe `new_for_section_key_info` now can be renamed as `new` function as it becomes the only constructor?

Reply:
Could be but the name of the struct is more generally useful, so it seem sensible to have a named constructor. Not sure.

Reply:
right. Ok then.
up to you to decide change or not, won't make this as a blocking comment.  :)